sdk/python/kfp/components/_components.py

```diff
@@ -483,6 +483,7 @@ def _resolve_command_line_and_paths(
     argument_serializer: Callable[[str], str] = serialize_value,
     input_uri_generator: Callable[[str], str] = _generate_input_uri,
     output_uri_generator: Callable[[str], str] = _generate_output_uri,
+    input_value_generator: Optional[Callable[[str], str]] = None,
```

Comment:
Is there a place where this is specified?

Reply:
Yes, only from v2.dsl:

https://github.com/kubeflow/pipelines/blob/2dd8de3f6f7663cc6ef9207d901dd243ae169d74/sdk/python/kfp/v2/dsl/component_bridge.py#L190-L198